src/bundle/ui-dev/src/modules/common/tooltip-popup/tooltip.popup.component.js

```diff
@@ -65,15 +48,35 @@ const TooltipPopupComponent = (props) => {
 
 TooltipPopupComponent.propTypes = {
     title: PropTypes.string.isRequired,
+    subtitle: PropTypes.string,
     children: PropTypes.node.isRequired,
     visible: PropTypes.bool.isRequired,
     onClose: PropTypes.func,
+    onConfirm: PropTypes.func,
     showFooter: PropTypes.bool,
+    confirmLabel: PropTypes.string,
+    closeLabel: PropTypes.string,
+    confirmBtnAttrs: PropTypes.object,
+    closeBtnAttrs: PropTypes.object,
 };
 
 TooltipPopupComponent.defaultProps = {
+    subtitle: '',
     onClose: () => {},
+    onConfirm: () => {},
     showFooter: true,
```

Comment:
Imo showFooter should be by default false.
Since all of other props provided to buttons are not required.
Also Footer should be another component bc now  u can pass showFooter to true without passing button click handlers which create uselss footer

Reply:
`TooltipPopupComponent` is used only onec in MFU so probably we can remove props `showFooter`